.github/workflows/ci.yml

```diff
@@ -0,0 +1,111 @@
+name: Pipeline
+
+on:
+  push:
+    branches:
+      - test_task
+  pull_request:
+    branches:
+      - master
+  workflow_dispatch:
+    inputs:
+      registry:
+        description: "Registry"
```

Comment:
Add better description everywhere, your description should always explain for what this param is needed